sftp/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "sunset-sftp"
 version = "0.1.0"
-edition = "2024"
+edition = "2021"
```

Comment:
I think using edition 2024 is probably fine, feel free to bump the version to 1.85 (or later)

https://github.com/mkj/sunset/blob/8be5415f44a5ebe2f153933a2a62762f482d3177/.github/workflows/ci.yml#L16-L17